src/tentaclio/fs/remover.py

```diff
@@ -10,20 +8,9 @@
 __all__ = ["REMOVER_REGISTRY", "ClientRemover"]
 
 
-class Remover(Protocol):
+class Remover(ContextManager, Protocol):
```

Comment:
Inheriting from both ContextManager + Protocol seems to work now

Reply:
cool!